### PR TITLE
Wavefront class display() method: fix the handling of user-specified vmin, vmax arguments for phase map plot

### PR DESCRIPTION
**Pull request by neilzim**
Currently, when you display a phase map, user-specified vmin and vmax values are ignored. I added a few lines of logic so that when the user provides parameters that are not None, they override the defaults (-0.25, 0.25).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,6 +24,7 @@ Code by Marshall Perrin, Joseph Long, Ewan Douglas, with additional
 contributions from Anand Sivaramakrishnan, Remi Soummer, Christine Slocum,
 and others on the Astropy team.
 
+test edit
```
Comment (josePhoenix) [inline README.rst]: Could you remove this?

**Comment by josePhoenix** [comment]
Can you write a sentence or so for the docstring saying what happens in the default case (0, max(data) for intensity; -.25, .25 for phase it looks like?)

**Comment by josePhoenix** [comment]
Since the build with actual code changes passed, I'm going ahead and merging before Travis completes.

**Comment by mperrin** [comment]
I didn't have a chance to look at this before you merged it. A small suggestion would be to add to the docstring for vmin,vmax another sentence as follows:  "If the scale parameter is set to 'log', then the default vmin for intensity becomes 1e-6*intens.max() instead of 0."

**Pull request merged**